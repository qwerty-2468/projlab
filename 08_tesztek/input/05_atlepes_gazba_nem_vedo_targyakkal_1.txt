create room r1
create room r2
setroom r2 gas true
create student s
create tvsz tv
create airfresher a
create camembert c
create transistor tr
create rag r

neighbour r1 r2
addperson s r1
add tv s
add a s
add c s
add tr s
add r s

enter r2 s

status s