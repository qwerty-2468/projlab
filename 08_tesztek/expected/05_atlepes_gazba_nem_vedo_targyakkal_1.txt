
s
location: r2
itemsInHand: 
stunRemaining: 3
